web/src/engine/osk/src/views/oskView.ts

```diff
@@ -81,25 +79,15 @@ export interface EventMap {
    */
   hideRequested: (key: KeyElement) => void;
 
-  /**
-   * This event is raised when the OSK's 'config' button is clicked.
-   * Adding a listener for the event will cause the 'config' button to be displayed for
-   * FloatingOSKView instances.
-   */
-  showConfig: () => void;
-
-  /**
-   * This event is raised when the OSK's 'help' button is clicked.
-   * Adding a listener for the event will cause the 'help' button to be displayed for
-   * FloatingOSKView instances.
-   */
-  showHelp: () => void;
-
   /**
    * Signals the special command to display the engine's version + build number.
    */
   showBuild: () => void;
 
+  // While the next two are near-duplicates of the legacy event `resizeMove`, these
```

Comment:
```suggestion
  // While the next two are near-duplicates of the legacy event `resizemove`, these
```

Is that right? (Could we update resizemove to accept an optional promise?)

Reply:
tl;dr:  Unfortunately, we can't do so and maintain both backward-compatibility and functionality.

While it's not super-clear from https://help.keyman.com/developer/engine/web/17.0/reference/events/osk.resizemove, the legacy event unfortunately operates from a 'past tense' perspective - it only gets raised when the operation is completed.  The new variant gets raised when the operation starts, with the `Promise` serving as the signal of completion.

So... if we were to accept an "optional Promise", we'd either have to change legacy behavior or return pre-resolved Promises... the latter of which wouldn't help with the engine's related focus-management.